mbf_abstract_nav/src/move_base_action.cpp

```diff
@@ -230,10 +230,7 @@ void MoveBaseAction::actionExePathFeedback(
       {
         mbf_msgs::MoveBaseResult move_base_result;
         move_base_result.outcome = mbf_msgs::MoveBaseResult::OSCILLATION;
-        if(recovery_enabled_)
```

Comment:
Why removing the `recovery_enabled` option?

Reply:
We don't mention recovery behaviors on other failures message (planner/controller failures). So why to do on OSCILLATION?